Framework/PythonInterface/plugins/algorithms/VesuvioPeakPrediction.py

```diff
@@ -107,6 +107,23 @@ def PyExec(self):
 
                 vesuvio_params.addRow([temp, self._atomic_mass, self._debye_temp, kinetic, rms_momentum, rms_disp])
 
+        if self._model == "Classical":
+            vesuvio_params.addColumn("float", "RMS Momentum(A-1)")
+            vesuvio_params.addColumn("float", "Kinetic Energy(mEV)")
+
+            for temp in self._temperature:
+                if temp == 0:
```

Comment:
Just wanted to check if this is for the explicit removal of 0s for the sake of avoiding div 0 errors, or if it's checking for bounding and so may want to be `if temp < MIN_TEMPERATURE` instead